pkg/node/local.go

```diff
@@ -180,6 +182,27 @@ func StartLocalNode(
 		localDataExists = false
 	}
 
+	// set node configs
+	if nodeConfig == "" {
+		nodeConfig = "{}"
+	}
+	nodeConfig, err = utils.SetJSONKey(
+		nodeConfig,
+		config.PartialSyncPrimaryNetworkKey,
+		true,
+	)
+	if err != nil {
+		return err
+	}
+	nodeConfig, err = utils.SetJSONKey(
```

Comment:
Is `ProposerVMUseCurrentHeightKey` required here? I think it shouldn't be necessary with the ACP-151. If it is still required, could you add a comment explaining why?

Reply:
I just included it because it was not included previously and reminded the flag was asked to be set.
Will just not include as I am not pretending it is still required.